### ISSUE TITLE
FileFinder has no attribute find_module

### ISSUE THREAD

**Issue by ebachUTSA** [AnacletoLAB/grape#62]
File "...\Lib\site-packages\grape\__init__.py", line 83, in <module>
    import_all(locals())
  File "...\Lib\site-packages\grape\__init__.py", line 26, in import_all
    _loaded_module = _loader.find_module(
                     ^^^^^^^^^^^^^^^^^^^
AttributeError: 'FileFinder' object has no attribute 'find_module'

I've installed into a venv using pip. Python version 3.12. The github shows you have a try except to handle this which leads me to believe that whatever is out on pip is not representative of what is on github. I tried reinstalling from github with a pip install git+url but that generated an installation error I could not overcome:

ERROR: Could not find a version that satisfies the requirement ensmallen>=0.8.88 (from grape) (from versions: 0.0.1, 0.6.0, 0.6.1, 0.6.2, 0.6.3, 0.6.4, 0.6.5, 0.6.6, 0.7.0.dev19, 0.8.0, 0.8.1, 0.8.2, 0.8.25, 0.8.26, 0.8.27, 0.8.28, 0.8.29, 0.8.36, 0.8.42, 0.8.43, 0.8.44)
ERROR: No matching distribution found for ensmallen>=0.8.88

**Comment by ahmedsalim3** [comment]
Hi, I encountered the same issue
I resolved it by creating a new virtual env and installing Python 3.9 in it

**Comment by ebachUTSA** [comment]
Thanks, I'm trying to avoid using < 3.12 due to concurrent steps in the analytic pipeline that leverage 3.12. I'm currently using a different graph analytic process but I am hoping this gets updated so I can incorporate it and do a speed comparison.

**Comment by zommiommy** [comment]
Yup, We will fix it. I will have time to look into it after the 5 of September.

**Comment by ebachUTSA** [comment]
Awesome, I'm excited to test it out. I have several large datasets I want to use this on and Gephi and networkx can't handle them very well.
